PhpStorm Protocol (Win)/run_editor.js

```diff
@@ -63,6 +63,14 @@ if (match) {
     shell.AppActivate(settings.window_title);
 }
 
+function isToolboxInstalled() {
+	var shell = new ActiveXObject('WScript.Shell'),
+        appDataLocal = shell.ExpandEnvironmentStrings("%localappdata%"),
+        toolboxDirectory = appDataLocal + '\\JetBrains\\Toolbox\\apps\\PhpStorm\\ch-0';
```

Comment:
Is the toolbox installed into a `ch-0` folder?

Reply:
It's the same folder used in the `configureToolboxSettings` function.

Reply:
To really answer your question @aik099, Toolbox is installed in %localappdata%\Jetbrains\Toolbox, but this also checks that PHPStorm is installed and that can prevent some errors.

Reply:
Thank you.